src/pages/page-router.ts

```diff
@@ -82,8 +110,10 @@ export class PageRouter extends LitElement {
   }
 
   static styles = css`
-    * {
-      flex: 1;
+    recording-page {
+      width: 100%;
+      height: 100%;
+      align-items: center;
```

Comment:
Are you trying to align the items inside the `recording-page`? That seems like something that should be defined inside `recording-page` ?